client/src/App.css

```diff
@@ -1,37 +1,92 @@
 .app {
   text-align: center;
-  background-color: black;
-  color: #66fcf1;
+  background-color: white;
 }
 
 .big-header {
   padding: 40px;
 }
-
+.filmBase {
```

Comment:
yes I forgot this, and I noticed the difference on style but because of fatigue I didnt look back and check. I hope I will better about timing and planning .